.circleci/config.yml

```diff
@@ -32,6 +32,35 @@ commands:
                     echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
                     echo 'nvm use 16' >> $BASH_ENV
 jobs:
+    aks_integration_tests:
+        machine:
+            docker_layer_caching: true
+            enabled: true
+            image: ubuntu-2004:202010-01
+        steps:
+            - checkout
+            - install_python_requests
+            - azure-cli/install
+            - run:
+                command: mkdir -p /tmp/logs/test/integration/aks
+                name: Create temp dir for logs
+            - run:
+                command: |
+                    export NVM_DIR="/opt/circleci/.nvm"
+                    [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+                    nvm install v16
+                    npm install
+                    export KUBERNETES_MONITOR_IMAGE_NAME_AND_TAG=$(./scripts/circleci-jobs/setup-integration-tests.py)
+                    .circleci/do-exclusively --job ${CIRCLE_JOB} npm run test:integration:aks:yaml
```

Comment:
Is it OK to block on other PRs? Should we make it like the existing EKS and OS4 tests - run only on merge to staging?

Reply:
Fixed :) I've changed it for debug purpose and forgot 🤦